README.md

```diff
@@ -120,6 +121,30 @@ Example of multiple slot filling used in the same intent:
 <img src="./screenshots/slot-filling-2.png" width="auto" height="auto"/>
 </div>
 
+## Docker Compose
+
+A docker-compose receipt is available to allow a quick-start easily. Just execute:
+
+**Start:**
+
+```docker
+docker-compose up
+```
+
+**Start detached:**
+
+```docker
+docker-compose up -d
+```
+
+**Stop:**
+
+```docker
+docker-compose down
+```
+
+It will start the Mongo database, [NLP.js](http://localhost:3000 "NLP.js app at localhost") application and a [Mongo Express](http://localhost:8081 "Mongo-Express at localhost") to allow inspect the collections.

```

Comment:
I think that this link should fail:
Suggestion:
[NLP.js](http://localhost:3000) "NLP.js app at localhost"

Reply:
and
[Mongo Express](http://localhost:8081) "Mongo-Express at localhost"

Reply:
My bad, these links are ok. Go to merge

Reply:
These are links to reach services executed locally with docker-compose and, due to the base-configuration, these are the correct ports. I think it's ok.